### DISCUSSION TITLE
Frontend Initiatives Meeting: February 17, 2021

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#227] [Meetings] [answered]
When: February 3, 2021 @ 15:00 CET
Where: Remote

# Agenda

## Updates

- General
- Working groups
- Others?

## Topics

- Add network shim to apps that already use cypress (utils?) so we can run tests on GH (@Mohammer5)
- Start a discussion about component accessibility in `@dhis2/ui` (@paschalidi)
- PR previews short demo on the solution (@paschalidi)
- `styled-jsx` [bug in 3.4.x](https://github.com/vercel/styled-jsx/issues/695)

## Questions

- When will we be able to deploy independently applications? @paschalidi

**Comment by Mohammer5** [comment]
* Add network shim to apps that already use cypress (utils?) so we can run tests on GH

**Comment by KaiVandivier** [comment]
Update on storybook renovation + short show & tell 🙂

**Comment by amcgee** [comment, marked as answer]
Meeting notes are here https://github.com/dhis2/notes/blob/master/decisions/2021/02/17-fe-meeting.md
